vue/test/components/input-color/input-color.test.js

```diff
@@ -0,0 +1,10 @@
+const assert = require("assert");
+const base = require("../../base");
+
+describe("InputColor", () => {

```

Comment:
Please improve this by adding tests where you change the value in the input and checking that the color changes.